devresources/urls.py

```diff
@@ -33,10 +33,10 @@
     path('', include('core.urls')),
 ]
 
-if settings.DEBUG==True:
+if settings.DEBUG:
```

Comment:
Do not need to add `DEBUG==True`, it works same as only writing `DEBUG` in `if` condition.

Reply:
@hamhaingaurav Isn't that what I did in my code?